README.md

```diff
@@ -64,6 +64,16 @@ also build the documentation locally:
     cd listenbrainz-server/docs
     pip install -r requirements.txt
     make clean html
+    
+## Pushing test images using Github Actions
+
+1. Go to Actions -> Push deployment image or click [here](https://github.com/metabrainz/listenbrainz-server/actions/workflows/push-dev-image.yml).
+2. Select the branch and enter the docker image tag (version).
+![Screenshot (54)](https://user-images.githubusercontent.com/27751938/136417112-20184ff0-3140-425f-89fe-bae625b820a3.png)
```

Comment:
where is this image stored?

Reply:
This image is curently on github servers, I'll move it to the repo so that we can use it on RTD. We have an images folder under docs for this.